src/lib/stores/filters.ts

```diff
@@ -2,7 +2,35 @@ import type {
   WorkflowFilter,
   WorkflowSort,
 } from '$lib/models/workflow-filters';
-import { writable } from 'svelte/store';
+import { writable, derived, get } from 'svelte/store';
+import { page } from '$app/stores';
+import type { StartStopNotifier } from 'svelte/store';
 
-export const workflowFilters = writable<WorkflowFilter[]>([]);
-export const workflowSorts = writable<WorkflowSort[]>([]);
+const query = derived([page], ([$page]) => $page.url.searchParams.get('query'));
+
+const parameters = derived([query], ([$query]) => {
+  return {
+    query: $query,
+  };
+});
+
+const updateFilters: StartStopNotifier<WorkflowFilter[]> = (set) => {
+  return parameters.subscribe(({ query }) => {
+    if (!query && get(workflowFilters).length) {
+      // Clear filters if there is no query
```

Comment:
> the query is cleared but the UI still shows the filters applied

Looks like this is an existing bug 😞 I've added some updates to the `filters` store to clear filters and sorts if no query exists in the page url. 

This should cover both use cases of clicking the workflows icon and clicking the logo for recent workflows.

Reply:
👍 nice